caffe2/operators/fully_connected_op.cc

```diff
@@ -237,3 +243,23 @@ REGISTER_GRADIENT(FCTransposed, GetFCGradient);
 } // namespace
 
 } // namespace caffe2
+
+C10_REGISTER_CAFFE2_OPERATOR_CPU(
+    FC,
+    "_caffe2::FC("
+    "    Tensor[] inputs_list,"
```

Comment:
why do you bind this op and other ops with tensor list as input? why not create individual inputs?

Reply:
Fixed